File: framework/src/main/java/wulinpeng/com/framework/base/ui/image/imageloader/IImageLoaderStrategy.java
package wulinpeng.com.framework.base.ui.image.imageloader;

import android.content.Context;

/**
 * @author wulinpeng
 * @datetime: 17/4/9 下午8:14
 * @description:
 */
public interface IImageLoaderStrategy {

    public void load(Context context, ImageLoadEntity imageLoadEntity);
}

File: framework/src/main/java/wulinpeng/com/framework/base/ui/image/imageloader/IImageLoaderStrategy.kt
package wulinpeng.com.framework.base.ui.image.imageloader

import android.content.Context

/**
 *
 * @author wulinpeng
 * @datetime: 2019/4/6 8:37 PM
 * @description:
 */
interface IImageLoaderStrategy {
    fun load(context: Context, imageLoadEntity: ImageLoadEntity)
}